tests/test_hosting_commands.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from syncano_cli.hosting.utils import HostingCommands
+from syncano_cli.main import cli
+from tests.base import BaseCLITest
+
+
+class HostingCommandsTest(BaseCLITest):
```

Comment:
BaseCLITest inherits from InstanceMixin:
```
class InstanceMixin(object):

    @classmethod
    def setUpClass(cls):
        super(InstanceMixin, cls).setUpClass()

        cls.instance = cls.connection.Instance.please.create(
            name='test_cli_i%s' % cls.generate_hash()[:10],
            description='IntegrationTest %s' % datetime.now(),
        )

    @classmethod
    def tearDownClass(cls):
        cls.instance.delete()
        super(InstanceMixin, cls).tearDownClass()
```

So it's a instance common for all test - and then I need to correct the publish test then :) Good catch.